### PR TITLE
remove arg -B in tiup

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12048
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qqqdan — APPROVED** [review]

**Review by nexustar — APPROVED** [review]

File: tiup/tiup-overview.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Components Manifest:
   use "tiup list" to fetch the latest components manifest
 
 Flags:
-  -B, --binary <component>[:version]   Print binary path of a specific version of a component <component>[:version]
+  --binary <component>[:version]   Print binary path of a specific version of a component <component>[:version]
```
Comment (Oreoxmt) [inline tiup/tiup-overview.md:74]:
```suggestion
      --binary <component>[:version]   Print binary path of a specific version of a component <component>[:version]
```

File: tiup/tiup-reference.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ You can use the `--help` command to get the information of a specific command. T
 
 ## Options
 
-### -B, --binary
+### --binary
```
Comment (shichun-0415) [inline tiup/tiup-reference.md:23]: Do we need to remove `-B/-` in Lines 27 and 28 as well?

Comment (Oreoxmt) [reply tiup/tiup-reference.md:23]: Done https://github.com/pingcap/docs/pull/11362/commits/0ba3745dd419b660a415c3670cf742d102d55028

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**